.github/workflows/docker_release.yml

```diff
@@ -107,3 +107,21 @@ jobs:
             UI_RELEASE=${{ steps.manifest.outputs.UI_RELEASE }}
             BUILD_VERSION=${{ steps.build_tag_generator.outputs.RELEASE_TAG }}
             GIT_REF=${{ github.ref }}
+
+      - name: Ensure manifest.json exists
+        run: |
+          if [ ! -f manifest.json ]; then
+            echo "manifest.json not found in repo root - generating via manifestgen.sh"
+            ./manifestgen.sh || true
+          fi
+
+      - name: Upload manifest.json to GitHub Release
+        if: github.event_name == 'release'
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ./manifest.json
+          asset_name: manifest.json
+          asset_content_type: application/json
```

Comment:
Why is this not a separate workflow?

I also think the generating a manifest should not be done as part of the workflow because it might not match the branch. We have a tag branch per release, and it's better to make sure the release matches that imo.